ibmdbpy/base.py

```diff
@@ -1594,9 +1600,10 @@ def _upper_columns(self, dataframe):
         """
         Put every column name of a Pandas DataFrame in upper case.
 
-        Arguments
-        ---------
+        Parameters
+        ----------
         dataframe: DataFrame
```

Comment:
```suggestion
        dataframe : DataFrame
```

https://numpydoc.readthedocs.io/en/latest/format.html, says " The colon must be preceded by a space, or omitted if the type is absent."